AppCreationScripts/apps.json

```diff
@@ -1,33 +1,41 @@
 {
-  "Sample": {
-    "Title": "Calling Microsoft Graph from an iOS app",
-    "Level": 400,
-    "Client": "iOS"
-  },
-  "AppRegistrations": [
-    {
-      "x-ms-id": "active-directory-ios-swift-native-v2",
-      "x-ms-name": "ios-swift-native-v2",
-      "x-ms-version": "2.0",
-      "replyUrlsWithType": [
-        {
-          "url": "msal{AppId}://auth",
-          "type": "InstalledClient"
-        }
-      ],
-      "requiredResourceAccess": [
-        {
-          "x-ms-resourceAppName": "Microsoft Graph",
-          "resourceAppId": "00000003-0000-0000-c000-000000000000",
-          "resourceAccess": [
-            {
-              "id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
-              "type": "Scope",
-              "x-ms-name": "user.read"
-            }
-          ]
-        }
-      ]
-    }
-  ]
-}
+	"Sample": {
+		"Title": "Calling Microsoft Graph from an iOS app",
+		"Level": 400,
+		"Client": "iOS"
+	},
+	"AppRegistrations": [{
+		"x-ms-id": "active-directory-ios-swift-native-v2",
+		"x-ms-name": "ios-swift-native-v2",
+		"x-ms-version": "2.0",
+		"replyUrlsWithType": [{
+			"url": "msal{AppId}://auth",
```

Comment:
Does {AppId} translate to bundleId?

Reply:
```{AppId}``` should be replaced by the one user entered on the portal. Let me double check with Tad since this line of code has been here for a long time and I simply refactored the format

Reply:
Yeah, this looks like an outdated format as well. We used to have this default format before (msal<clientid>://auth), but it is now deprecated to support broker with bundleId (msauth.<bundleId>://auth).

Reply:
Updated